app/src/renderer/connectors/lcdClient.js

```diff
@@ -199,19 +199,11 @@ const Client = (axios, localLcdURL, remoteLcdURL) => {
       return req(`POST`, `/gov/proposals/${proposalId}/deposits`, true)(data)
     },
     getGovernanceTxs: async function(address) {
-      let [depositerTxs, proposerTxs] = await Promise.all([
-        req(
-          `GET`,
-          `/txs?tag=action='submit-proposal'&tag=proposer='${address}'`,
-          true
-        )(),
-        req(
-          `GET`,
-          `/txs?tag=action='deposit'&tag=depositer='${address}'`,
-          true
-        )()
+      let [depositorTxs, proposerTxs] = await Promise.all([
```

Comment:
👍

Reply:
Minor: if you want we can write it simlarly as line 84 and store a variable less

Reply:
Done.